controllers/namespacescope/namespacescope_controller.go

```diff
@@ -0,0 +1,250 @@
+//
+// Copyright 2021 IBM Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package namespacescope
+
+import (
+	"context"
+	"time"
+
+	gset "github.com/deckarep/golang-set"
+	"github.com/pkg/errors"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/discovery"
+	"k8s.io/klog"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	nssv1 "github.com/IBM/ibm-namespace-scope-operator/api/v1"
+
+	operatorv1alpha1 "github.com/IBM/operand-deployment-lifecycle-manager/api/v1alpha1"
+	"github.com/IBM/operand-deployment-lifecycle-manager/controllers/constant"
+	deploy "github.com/IBM/operand-deployment-lifecycle-manager/controllers/operator"
+	"github.com/IBM/operand-deployment-lifecycle-manager/controllers/util"
+)
+
+// Reconciler automagically updates NamespaceScope CR with the proporly namespace
+type Reconciler struct {
+	*deploy.ODLMOperator
+}
+
+// ReconcileOperandRequest reads that state of the cluster for OperandRequest object and update NamespaceScope CR based on the state read
+func (r *Reconciler) ReconcileOperandRequest(req ctrl.Request) (_ ctrl.Result, reconcileErr error) {
+	// Creat context for the OperandBindInfo reconciler
```

Comment:
I did not understand this comment, why this is the `OperandBindInfo reconciler`?